Source/WebKit/WebProcess/GPU/graphics/RemoteRenderingBackendProxy.messages.in

```diff
@@ -23,7 +23,7 @@
 #if ENABLE(GPU_PROCESS)
 
 messages -> RemoteRenderingBackendProxy NotRefCounted {
-    DidCreateImageBufferBackend(WebKit::ImageBufferBackendHandle handle, WebCore::RenderingResourceIdentifier renderingResourceIdentifier)
+    DidCreateImageBufferBackend(WebKit::ImageBufferBackendHandle handle, WebCore::RenderingResourceIdentifier renderingResourceIdentifier, bool didFailToCreateIOSurface)
```

Comment:
I think you should send a RenderingMode back (like `RenderingMode renderingModeOfAllocatedBuffer`).

Reply:
Fixed.